routes/apiRoutes.js

```diff
@@ -89,9 +89,11 @@ module.exports = function(app) {
             return;
         }
 
+        var newSellerId = getRandomString();
+
         const data = req.body;
 
-        const itemProps = JSON.parse(data.properties);
+        const itemProps = {};//JSON.parse(data.properties);
```

Comment:
This was a temporary change. so if we leave it in we should add another task to revert it and add error checking